lib/rubocop/cop/style/negated_if.rb

```diff
@@ -4,14 +4,38 @@ module RuboCop
   module Cop
     module Style
       # Checks for uses of if with a negated condition. Only ifs
-      # without else are considered.
+      # without else are considered. There are three different styles:
+      #
+      # both - enforces `unless` for `prefix` and `postfix` conditionals
```

Comment:
You should add `# good` and `# bad` and `@example` annotations to the code examples.